api/src/main/java/edu/cornell/mannlib/vitro/webapp/rdfservice/impl/jena/RDFServiceJena.java

```diff
@@ -201,8 +201,8 @@ private void removeBlankNodesWithSparqlUpdate(Dataset dataset, Model model, Stri
         log.debug("blank node model size " + blankNodeModel.size());
 
         if (blankNodeModel.size() == 1) {
-            log.warn("Deleting single triple with blank node: " + blankNodeModel);
-            log.warn("This likely indicates a problem; excessive data may be deleted.");
+            log.debug("Deleting single triple with blank node: " + blankNodeModel);
+            log.debug("This likely indicates a problem; excessive data may be deleted.");
```

Comment:
```suggestion
            log.debug("This may indicate a problem; excessive data may be deleted.");
```
Remove some of the doom and gloom if the warning is most likely to be triggered by normal operation.